src/components/shared/Fields/InputStatus/InputStatus.tsx

```diff
@@ -1,19 +1,26 @@
 import { MessageDescriptor, useIntl } from 'react-intl';
 import React from 'react';
+import { FieldError, FieldErrorsImpl, Merge } from 'react-hook-form';
 
-import { isNil } from 'lodash';
 import { SimpleMessageValues } from '~types';
 import { getMainClasses } from '~utils/css';
+import { isNil } from '~utils/lodash';
 
 import { InputComponentAppearance as Appearance } from '../Input';
+
 import styles from './InputStatus.css';
 
 interface Props {
   /** Appearance object */
   appearance?: Appearance;
 
   /** Error text (if applicable) */
-  error?: string | MessageDescriptor;
+  error?:
+    | string
+    | MessageDescriptor
+    | FieldError
+    | Merge<FieldError, FieldErrorsImpl<any>>
```

Comment:
These types are HookForm types, no? But this isn't the HookForm status component. Also, you should be able to cast the error as a `Message` type since we control what the error message is.

Reply:
You are correct. I didn't notice that a HookForm status component already existed